src/color.temper.md

```diff
@@ -30,13 +31,13 @@ TODO Some consistent representation for opacity?
 
 While `rows` is public, it's sometimes nice to skip through it.
 
-    public get(i: Int, j: Int): Float64 | Bubble { rows[i, j] }
+      public get(i: Int, j: Int): Float64 | Bubble { rows[i, j] }
```

Comment:
I don't remember if I left these dedented on purpose or not. But I can see the argument for normalizing them either way.